src/__tests__/Converter.test.ts

```diff
@@ -47,24 +47,23 @@ describe('NavDrawer component', () => {
 
 		//find the first example option and click it
 		const firstExample = await openProjectWidget.findElement(
-			By.css('li.MuiListItem-root:nth-child(1) > div:nth-child(1)')
+			By.xpath("//div[contains(text(),'Proton pencil beam in water')]")
 		);
```

Comment:
This technically now finds ony div with text 'Proton pencil beam in water' in it. 
it would be better to give some sort of id or label to this list
![image](https://github.com/yaptide/ui/assets/26445545/d28fbdc2-b324-4e6d-8719-0b1ac0e90ab2)
and find list element with value 0
Lastly maybe use getText method and check div contents are correct?
something like that
![image](https://github.com/yaptide/ui/assets/26445545/1346630e-d0d4-49c1-8f18-06d939fd81c1)
```tsx
<ListItem
    disablePadding
    key={
	    example?.inputJson?.project?.title ??
	    'Example_' + idx.toString()
    }
    value={idx}
    aria-labelledby={`example-btn-${idx}`}
    aria-selected={exampleIndex === idx}
    onClick={() => setExampleIndex(idx)}>
    <ListItemButton
	    id={`example-btn-${idx}`}
	    selected={exampleIndex === idx}>
	    {example?.inputJson?.project?.title ??
		    'Example_' + idx.toString()}
    </ListItemButton>
</ListItem>
```
And for the tests
```ts
//find the first option in the list
//it has value=0 and is li element
const firstExample = await examplesList.findElement(
   By.xpath("//li[@value = '0']")
);

//check if the first example is "Proton pencil beam in water"
//text is located in element with id corresponding to aria-labelledby of the list element
const exampleLabelId = await firstExample.getAttribute("aria-labelledby");
const exampleLabel = await openProjectWidget.findElement(By.id(exampleLabelId));
expect(await exampleLabel.getText()).toBe("Proton pencil beam in water");

//click the first example
await firstExample.click();
```

Reply:
alright